assembly/Cargo.toml

```diff
@@ -15,10 +15,17 @@ rust-version = "1.60"
 bench = false
 doctest = false
 
+[[bench]]
+name = "sha256"
```

Comment:
I'd probably change the name to "script_compilation" or something similar.

Reply:
Done!